### DISCUSSION TITLE
The card is very cool but wondering about a small issue

### DISCUSSION THREAD

**Discussion by MidnightBlue2049** [markadev/AppleII-VGA#57] [Q&A]
Hi together,
Last week a bought an Apple II VGA card and I am very happy with it. 
I am a Apple II newbie (altough I bought the first one 1987)
A small issue is left for a 100% happyness but I do not know if I am wrong with it.
The shown display content is very sharp but I am observing odd colors in fonts where the characters are tight to each other placed or if the font is more fancy (see attached pic). In other games screens I get a small odd colored shadow between the characters.
Is that issue resolvable?
![20241114_172527-EDIT](https://github.com/user-attachments/assets/984c0677-415d-4504-b62f-6175d231fcdf)

**Comment by markadev** [comment]
Hi there! That violet/green color fringing is typical with Apple II graphics due to the way that the Apple II generates color.  The Apple II doesn't actually generate a color video signal like you'd think of it in modern tech, it generates a monochrome pattern and each pixel has a color based on its column on the display.  Close pixels then sort of blur together to make larger areas of color like you see in the "W" and "N" characters and directly adjacent pixels will blend together to make white areas.  This is called NTSC artifact color.

Back in the 1980's, for software that makes extensive use of high resolution monochrome graphics where you wouldn't want this color fringing then you would use a monochrome monitor (or flip a switch to disable color on your color monitor).  The same effect can be done with the VGA card using the soft monochrome mode that can be enabled from the configuration disk.

**Reply by MidnightBlue2049** [reply]
Thank you for your fast and clear answer.
Nevertheless I am a great fan of your excellent work.
For me it is amazing to see that sharp screen and the beautiful colors of my Apple IIe every time I switch it on :-D
